ecosystem/platform/server/app/controllers/projects_controller.rb

```diff
@@ -32,8 +32,11 @@ def index
   # GET /projects/1
   def show
     @project = Project.find(params[:id])
-    head :not_found unless @project.verified
-    head :forbidden if @project.user_id != current_user&.id && !@project.public
+
+    if @project.user_id != current_user&.id
+      return head :not_found unless @project.verified
```

Comment:
could you add a `|| current_user&.is_root`?

Reply:
Done.